mu4/scenes/playback/internal/playbackcontroller.cpp

```diff
@@ -86,29 +92,41 @@ void PlaybackController::togglePlay()
     }
 
     if (isPlaying()) {
-        pause();
+        stop();
     } else {
         play();
     }
 }
 
 void PlaybackController::play()
 {
-    auto notation = globalContext()->currentNotation();
-    IF_ASSERT_FAILED(notation) {
+    IF_ASSERT_FAILED(m_notation) {
         return;
     }
 
-    auto stream = notation->playback()->midiStream();
+    auto stream = m_notation->playback()->midiStream();
     audioPlayer()->setMidiStream(stream);
+
+    int tick = m_notation->playback()->playPositionTick();
```

Comment:
I think it's better to use RetVal here

If we don't have a masterScore, then we'll try to play non-existing score from the 0 tick

Reply:
fixed